src/components/Flow/Flow.css

```diff
@@ -0,0 +1,22 @@
+@import "@folio/stripes-components/lib/variables";
+
+.flowList {
+  margin-bottom: 1rem;
+  margin-right: calc(var(--gutter-static-one-third) * -1);
```

Comment:
What is `gutter-static-one-third`?

Reply:
It's the amount of padding Rasmus said to use here. He also advised me to counter it with a negative margin so it still lined up with something outside that region entirely, I think the current app icon or similar.

Reply:
Ah, sorry, I was unclear. What I meant is, where is `--gutter-static-one-third` defined — where does client code get it from? And how are client coders supposed know to do this?

(More generally, shouldn't the stripes-components components be set up so that their users don't have to involve themselves in such incantations?)

Reply:
https://github.com/folio-org/stripes-components/blob/master/lib/variables.css

But it should be more explicitly public:
https://issues.folio.org/browse/STCOM-528

And if you can get the components to do it, you should because that's the goal. If you can't and have to resort to app level CSS, the more you rely on the CSS variables for such constants the more likely it is things will fit in with the overall design as it evolves.

Documented here:
https://github.com/folio-org/stripes-components/blob/master/docs/CSSinStripes.md#css-variables

Reply:
Thanks, this is pretty good. But if ever a CSS desperately needed comments, it's this one. When, for example, should one use {{--color-text-p2}}? I have no idea.

Reply:
Thanks, this is pretty good. But if ever a CSS desperately needed comments, it's this one. When, for example, should one use `--color-text-p2`? I have no idea.